docs_src/microservices/support/logging/Ch-Logging.md

```diff
@@ -60,44 +61,15 @@ by EdgeX Foundry Logging Service: file or MongoDB.
 
 ## Configuration Properties
 
+The following configuration properties are specific to the support-logging service. Please refer to the general Configuration documentation for configuration properties common across all services.
```

Comment:
Make general Configuration documentation a link back to the general Configuration documentation